scripts/check_configs.ts

```diff
@@ -426,6 +443,35 @@ function isValidMarketPromo(marketPromo: {[marketId: string]: MarketPromo}, netw
   return true;
 }
 
+function isValidTransferBanner(transferBanner: TransferBanner, network: CarbonSDK.Network): boolean {
+  const { temporary_disabled_transfer_tokens, temporary_disabled_bridges } = transferBanner;
+  if (Object.keys(temporary_disabled_transfer_tokens).length > 0) {
+    Object.keys(temporary_disabled_transfer_tokens).map((key) => {
+      const { start, end } = temporary_disabled_transfer_tokens[key];
+      const startTime = new Date(start);
+      const endTime = new Date(end);
+      if (endTime < startTime) {
+        console.error(`ERROR: ${network}.json has invalid end time (${end}) is before start time (${start}) for token denom ${key}.`);
+        return false;
+      }
+    });
+  }
+
+  if (Object.keys(temporary_disabled_bridges).length > 0) {
+    Object.keys(temporary_disabled_bridges).map((key) => {
+      const { start, end } = temporary_disabled_bridges[key];
+      const startTime = new Date(start);
+      const endTime = new Date(end);
+      if (endTime < startTime) {
+        console.error(`ERROR: ${network}.json has invalid end time (${end}) is before start time (${start}) for bridge address ${key}.`);
```

Comment:
Pls revise the message to `ERROR: ${network}.json has an invalid end time (${end}) as it is before start time (${start}) for bridge ${key}.`